src/tests/setup.js

```diff
@@ -0,0 +1,3 @@
+/**
+ * Code in this file will run before each test.
+ */
```

Comment:
Is this empty?

Reply:
Yep. its empty but could be useful in the future. Can be removed if necessary. I just copied it from our repo